packages/auth-sdk/user.tsx

```diff
@@ -36,7 +37,10 @@ export function useAuthenticatedUser() {
       userKeys.user,
       () => {
          const responsePromise = fetchAuthenticatedUser(apiClient).catch(
-            () => null
+            (e) => {
+               Sentry.captureException(e);
+               return null;
+            }
```

Comment:
The ifixit api client already does error handling for us. The only way we'd catch this exception is if the schema parsing threw.

Why not use `safeParse` in `fetchAuthenticatedUser` and return `null` if the parsing failed? We could also report a more specific exception like

```ts
   Sentry.captureMessage('User schema parsing failed', {
      level: 'debug',
      extra: {
         userSchema,
         payload,
      },
   });
```

Reply:
I don't think there is a significant difference between your proposal and what is implemented, but we can swap in safeParse for the sake of less try catches.

Reply:
I agree, I don't think there's a functional difference. My intent was to reduce try/catch's where we can, and to suggest that we report the debug error in fetchAuthenticatedUser